terraform/24-tenants/tenant.tf

```diff
@@ -3,9 +3,23 @@ resource "octopusdeploy_tenant" "tenant_team_a" {
   description = "Test tenant"
   tenant_tags = ["tag1/a", "tag1/b"]
   depends_on = [octopusdeploy_tag.tag_a, octopusdeploy_tag.tag_b]
+}
+
 
-  project_environment {
-    environments = [octopusdeploy_environment.test_environment.id, octopusdeploy_environment.development_environment.id, octopusdeploy_environment.production_environment.id]
-    project_id   = octopusdeploy_project.deploy_frontend_project.id
-  }
+resource "octopusdeploy_tenant_project_environment" "team_a_test_environment" {
```

Comment:
:heart: This solves so many problems.

Reply:
Just checking in to see why this hasn't been merged / released yet?

Is there any chance it might be in the next 24 hours?

Thank you! :-)